### PR TITLE
Merge 2019-10-01 0b087d9107

### PR DESCRIPTION
**Pull request by dawidd6**
Merge Homebrew/homebrew-core into Homebrew/linuxbrew-core

+ [ ] aircrack-ng
+ [ ] ccls
+ [ ] fftw
+ [ ] gcc
+ [ ] hdf5
+ [ ] jmeter
+ [ ] krakend
+ [ ] lapack
+ [ ] pumba

### CODE REVIEW

File: Formula/fftw.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ class Fftw < Formula
 
   bottle do
     cellar :any
-    sha256 "35006ccac20707ac44c02bc93f29b37899af307ea18d2d52dcea86ac1f80730b" => :x86_64_linux
```
Comment (dawidd6) [inline Formula/fftw.rb]: There were actually 2 linux SHAs, so I checked bintray and latest bottle had the other checksum, so I left it.

Comment (iMichka) [reply Formula/fftw.rb]: Ok for me, weird. Probably a merge mistake at one point or another.

**Comment by iMichka** [comment]
```
Error: Failed to import: /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/aircrack-ng.rb
aircrack-ng: /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/aircrack-ng.rb:9: syntax error, unexpected <<
<<<<<<< HEAD
  ^
/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/aircrack-ng.rb:11: syntax error, unexpected ===, expecting keyword_end
=======
   ^
/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/aircrack-ng.rb:13: syntax error, unexpected >>, expecting keyword_end
>>>>>>> aircrack-ng: update 1.5.2_1 bottle.
  ^
/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/aircrack-ng.rb:13: syntax error, unexpected ':', expecting keyword_end
>>>>>>> aircrack-ng: update 1.5.2_1 bottle.
                    ^
/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/aircrack-ng.rb:13: unexpected fraction part after numeric literal
>>>>>>> aircrack-ng: update 1.5.2_1 bottle.
                               ^
```

I'll redo that merge in smaller steps.

**Comment by iMichka** [comment]
Weird, as there are no conflict markers in the PR ... I think test bot does not like pull request with merge commits.

**Pull request closed without merging**